bin/src/upload.dart

```diff
@@ -77,9 +77,12 @@ ArgParser _initializeParser() => new ArgParser(allowTrailingOptions: true)
   ..addOption("token",
       help: "Token for coveralls", defaultsTo: Platform.environment["test"])
   ..addOption("workers", help: "Number of workers for parsing", defaultsTo: "1")
+  ..addOption("packages",
+      help: 'Where to find the packages file, that is, "package:..." imports.',
+      defaultsTo: ".packages")
   ..addOption("package-root",
-      help: 'Where to find packages, that is, "package:..." imports.',
-      defaultsTo: "packages")
+      help: 'Ignored/Deprecated. Package directories are no longer supported.',
+      defaultsTo: ".packages")
```

Comment:
Same spurious rename as before.  I think you want to leave this line as 'packages' not '.packages', not that it matters hugely since its ignored.